examples/benchmark_problems/simple_tests/cubic-fb.dat

```diff
@@ -0,0 +1,8 @@
+# FitBenchmark Problem
```

Comment:
To be consistent with the Neutron and Muon problem sets shouldn't we use a txt file extension for the new simple tests?

Reply:
fixed